src/Lib/Twig/Loader.php

```diff
@@ -138,12 +138,12 @@ private function getFilename($name)
             $paths = $this->getPaths($scope);
             foreach ($paths as $path) {
                 $filePath = $path . $file;
-                if (file_exists($filePath)) {
+                if (is_file($filePath)) {
```

Comment:
This change was necessary as the test app now has `templates/layout` folder and the test `$loader->getCacheKey('layout')` returned false positive.